mungegithub/github/github.go

```diff
@@ -1266,54 +1296,57 @@ func (obj *MungeObject) CloseIssuef(format string, args ...interface{}) error {
 }
 
 // ClosePR will close the Given PR
-func (obj *MungeObject) ClosePR() error {
+func (obj *MungeObject) ClosePR() bool {
 	config := obj.config
-	pr, err := obj.GetPR()
-	if err != nil {
-		return err
+	pr, ok := obj.GetPR()
+	if !ok {
+		glog.Errorf("Error in ClosePR")
+		return false
 	}
 	config.analytics.ClosePR.Call(config, nil)
 	glog.Infof("Closing PR# %d", *pr.Number)
 	if config.DryRun {
-		return nil
+		return true
 	}
 	state := "closed"
 	pr.State = &state
 	if _, _, err := config.client.PullRequests.Edit(config.Org, config.Project, *pr.Number, pr); err != nil {
 		glog.Errorf("Failed to close pr %d: %v", *pr.Number, err)
-		return err
+		return false
 	}
-	return nil
+	return true
 }
 
 // OpenPR will attempt to open the given PR.
 // It will attempt to reopen the pr `numTries` before returning an error
 // and giving up.
-func (obj *MungeObject) OpenPR(numTries int) error {
+//CHECK
+func (obj *MungeObject) OpenPR(numTries int) bool {
 	config := obj.config
-	pr, err := obj.GetPR()
-	if err != nil {
-		return err
+	pr, ok := obj.GetPR()
+	if !ok {
+		glog.Errorf("Error in OpenPR")
+		return false
 	}
 	config.analytics.OpenPR.Call(config, nil)
 	glog.Infof("Opening PR# %d", *pr.Number)
 	if config.DryRun {
-		return nil
+		return true
 	}
 	state := "open"
 	pr.State = &state
 	// Try pretty hard to re-open, since it's pretty bad if we accidentally leave a PR closed
 	for tries := 0; tries < numTries; tries++ {
-		if _, _, err = config.client.PullRequests.Edit(config.Org, config.Project, *pr.Number, pr); err == nil {
-			return nil
+		if _, _, err := config.client.PullRequests.Edit(config.Org, config.Project, *pr.Number, pr); err == nil {
+			return true
 		}
-		glog.Warningf("failed to re-open pr %d: %v", *pr.Number, err)
+		glog.Warningf("failed to re-open pr %d", *pr.Number)
```

Comment:
This was a real bug... 
Can you re-add the `err` but also break that if line (https://github.com/kubernetes/contrib/pull/1357/files#diff-6ff804e5140f204c3e7068a3ba138ae5R1340) into two lines so `err` is valid here?